e2e/src/config/tests-config/accounts/account-manager.ts

```diff
@@ -48,72 +53,61 @@ abstract class AccountManager implements IAccountManager {
     return { account: whaleAccount, accountWallet: whaleTxManager };
   }
 
-  whaleAccount(accIndex?: number): Wallet {
+  whaleAccount(accIndex?: number): NonceManager {
     return this.selectWhaleAccount(accIndex).accountWallet;
   }
 
   async generateAccount(initialBalanceWei = etherToWei("10")): Promise<Wallet> {
     const accounts = await this.generateAccounts(1, initialBalanceWei);
-    return this.getWallet(accounts[0]);
+    return accounts[0];
   }
 
-  async generateAccounts(
-    numberOfAccounts: number,
-    initialBalanceWei = etherToWei("10"),
-    retryDelayMs = 1_000,
-  ): Promise<Wallet[]> {
+  async generateAccounts(numberOfAccounts: number, initialBalanceWei = etherToWei("10")): Promise<Wallet[]> {
     const { account: whaleAccount, accountWallet: whaleAccountWallet } = this.selectWhaleAccount();
 
     console.log(
       `Generating accounts: chainId=${this.chainId} numberOfAccounts=${numberOfAccounts} whaleAccount=${whaleAccount.address}`,
     );
 
     const accounts: Account[] = [];
+    const promises: TransactionResponse[] = [];
```

Comment:
Chore: can we name this as just `transactionResponses` ? it was not a list of `Promise` at all but just a list of object having a field of `Promise` type